app/src/androidTest/java/com/omen273/crossLingo/TrainingTest.kt

```diff
@@ -0,0 +1,83 @@
+package com.omen273.crossLingo
+
+import android.view.View
+import android.widget.TextView
+import androidx.test.espresso.Espresso.*
+import androidx.test.espresso.action.ViewActions
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.matcher.RootMatchers.isDialog
+import androidx.test.espresso.matcher.ViewMatchers.*
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.platform.app.InstrumentationRegistry
+import com.omen273.crossLingo.R.id
+import junit.framework.TestCase.assertEquals
+import org.hamcrest.Description
+import org.hamcrest.Matcher
+import org.hamcrest.TypeSafeMatcher
+import org.hamcrest.core.AnyOf.anyOf
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class TrainingTest: TestBaseClass() {
+
+    private fun loadTraining(
+        chooseTopics: List<Int> = listOf(0)
```

Comment:
Do you really need default initialization?

Reply:
removed